mass_mailing_editor_af/views/snippets_themes.xml

```diff
@@ -1115,6 +1115,54 @@
         </div>
     </template>
 
+    <!--textmodul med H3 och länkknapp 2 kolumner-->
+    <template id="s_mail_block_toppyta_h3_button" name="H3 och länkknapp 2 kolumner">
+        <div class="snippets_div" style="width: 600px; height:206px; background-color: white; padding: 23.5px 24px 24px 24px;">
+            <div style="display:inline-block ; width:260px; background-color: white;">
```

Comment:
- Add vertical-align: top; to inline CSS of this div to ensure the div always is placed at the top 
(default is bottom when the second div gets a greater height)